spec/lita/adapters/slack/rtm_connection_spec.rb

```diff
@@ -18,7 +18,7 @@ def with_websocket(subject, queue)
   let(:channel) { Lita::Adapters::Slack::SlackChannel.new('C2147483705', 'general', 1360782804, 'U023BECGF', metadata) }
   let(:metadata) { Hash.new }
 
-  let(:rtm_start_response) do
+  let(:rtm_connect_response) do
```

Comment:
Is this response still an accurate depiction of what we get from the new rtm_connect call? (Note that on 9/28/22 rtm_start started returning the response of rtm_connect which has less info

Reply:
I updated the IMs value to be `nil` to match the new API response.